doc/classes/File.xml

```diff
@@ -30,14 +30,14 @@
 			<return type="void">
 			</return>
 			<description>
-				Close the currently opened file.
+				Closes the currently opened file.
 			</description>
 		</method>
 		<method name="eof_reached" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
-				Return whether the file cursor reached the end of the file.
+				Returns whether the file cursor reached the end of the file.
```

Comment:
Prefer booleans over "whether":

    Returns [code]true[/code] if the file cursor has reached the end of the file.